returnn/torch/engine.py

```diff
@@ -1361,6 +1388,8 @@ def _print_process(
             info += ["%s %s" % (k, _format_score_value(v)) for k, v in eval_info.items()]
         if batch_size_info:
             info += ["%s %s" % (k, _format_score_value(v)) for k, v in batch_size_info.items()]
+        if padding_ratio is not None:
+            info += [f"pad {padding_ratio:.1%}"]
```

Comment:
The problem here per batch is: As discussed above, we should separate this per data key. But then printing this info per batch really makes it too spammy.

I currently don't really know a good solution for this. So I think I would just not print this per batch for now, and only report at the end of the epoch.